pkgs/applications/virtualization/spice-vdagent/default.nix

```diff
@@ -8,6 +8,11 @@ stdenv.mkDerivation rec {
     sha256 = "0n9k2kna2gd1zi6jv45zsp2jlv439nz5l5jjijirxqaycwi74srf";
   };
   NIX_CFLAGS_COMPILE = [ "-Wno-error=address-of-packed-member" ];
+  patches = [
+    # included in the next release
+    (fetchpatch { url = "https://gitlab.freedesktop.org/spice/linux/vd_agent/-/commit/7b0435ef66af088c1a1be20b6bc6b0fcb76e4e1a.patch"; sha256 = "<FIXME>";}) 
+    (fetchpatch { url = "https://gitlab.freedesktop.org/spice/linux/vd_agent/-/commit/9b8c0ebb9fb573e6ce3c5416371509f416503d0c.patch"; sha256 = "<FIXME>";}) 
```

Comment:
Just so you don't forget: these two hashes need to be set.

Also, you need to specify `fetchpatch` in the arguments at the top of the file in order to use it.

Also also, you can drop `timeout.diff` because of this change :)

Reply:
Yeah the `<FIXME>` part was meant as a place holder.